_layouts/base.html.haml

```diff
@@ -116,7 +88,8 @@
                   %a(href="https://www.optaplanner.org" rel="publisher" target="_blank" title="OptaPlanner") OptaPlanner
                 %li.divider
                 %li<
-                  %a(href="https://www.kiegroup.org/" target="_blank" title="Knowledge is everything group") KIE group
+                  -# set to http since we don't have yet a valide certificate
+                  %a(href="http://www.kiegroup.org/" target="_blank" title="Knowledge is everything group") KIE group
```

Comment:
uses http. Never use http, always https for our websites (external ones sometimes need http)
using http instead of https reduces our SEO rankings

Reply:
OK